docs/Learning-Environment-Design-Brains.md

```diff
@@ -23,7 +23,7 @@ The Brain Inspector window in the Unity Editor displays the properties assigned
     * `Vector Observation` 
     	* `Space Type` - Corresponds to whether the observation vector contains a single integer (Discrete) or a series of real-valued floats (Continuous).
     	* `Space Size` - Length of vector observation for brain (In _Continuous_ space type). Or number of possible values (in _Discrete_ space type).
-		* `Stacked Vectors` - The number of previous vector observations that will be stacked before being sent to the brain.
+		* `Stacked Vectors` - The number of previous vector observations that will be stacked before being sent to the brain. This results in the effective size of the vector observation being passed to the brain being: _Space Size_ x _Stacked Vectors_.
 	* `Visual Observations`	- Describes height, width, and whether to grayscale visual observations for the Brain.
```

Comment:
Can you add a note on internal brains needing to have the same stacked vector obs as used during training ?